src/devtools/client/webconsole/components/FilterBar/FilterBar.js

```diff
@@ -33,8 +33,6 @@ const SearchBox = createFactory(require("devtools/client/shared/components/Searc
 
 const PropTypes = require("prop-types");
 
-const disabledCssFilterButtonTitle = l10n.getStr("webconsole.cssFilterButton.inactive.tooltip");
```

Comment:
Should we take out this string from webconsole.properties as well?

Reply:
good call